soh/soh/Enhancements/bossrush.cpp

```diff
@@ -0,0 +1,468 @@
+﻿#include "bossrush.h"
+#include "functions.h"
+#include "macros.h"
+
+#include <array>
+#include <string>
+#include <vector>
+
+extern "C" {
+uint64_t GetUnixTimestamp();
+}
```

Comment:
I think this is just me being a derp and not remove this after doing some testing with it.

Reply:
Nope, this was definitely needed. It's because the define for `GAMEPLAYSTAT_TOTAL_TIME` needs it. I've handled it inside of OTRGlobals.h now which should be a bit cleaner, even though it's now the only function in there that has a extern "C" based on wether it's used in a .c or .cpp file. Lemme know if this is acceptable.

Reply:
Looks like Linux doesn't like `globals.h` being included due to similar duplicate definition errors that Rozelette ran into with his dynamic actor IDs PR. We have a fix for that there that will be merged soon. I'm thinking for now, just put it back how you had it and we'll revisit that later. Maybe add a todo comment.

Reply:
Looks like you found a different way around it. OTRGlobals is fine, accessible to other features that way.